src/core/utils/components/VirtualScroll.test.tsx

```diff
@@ -7,52 +7,38 @@ import { fireEvent, render } from '@testing-library/react';
 import React from 'react';
 import VirtualScroll from './VirtualScroll';
 
-const component = (
-  <div style={{ overflow: 'auto', height: 400 }} id='scroller'>
-    <VirtualScroll
-      itemsLength={1000}
-      itemRenderer={(index) => (
-        <div key={index} className='element' style={{ height: 40 }}>{`Element${
-          index + 1
-        }`}</div>
-      )}
-    />
-  </div>
-);
-
-const renderComponent = () => {
-  return render(component);
-};
-
-const renderEmptyComponent = () => {
-  return render(
-    <div style={{ overflow: 'auto', height: 400 }} id='scroller'>
-      <VirtualScroll
-        itemsLength={1000}
-        itemRenderer={(index) => <div key={index} className='element' />}
-      />
-    </div>,
-  );
-};
-
 beforeAll(() => {
   // return correct values for container 'scroller' and children
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  function getBoundingClientRectMock(this: Record<string, any>) {
-    if (Object.values(this)[0].memoizedProps.id === 'scroller') {
-      return { height: 400 } as DOMRect;
-    }
-    return { height: 40 } as DOMRect;
-  }
   jest
     .spyOn(HTMLElement.prototype, 'getBoundingClientRect')
-    .mockImplementation(getBoundingClientRectMock);
+    .mockImplementation(function (this: Record<string, any>) {
```

Comment:
Since when we are using functions? 😀

Reply:
oh please show me, how to do this with arrow func :)

Reply:
Ok, wow, it doesn't allow 😲